lib/rucio/core/heartbeat.py

```diff
@@ -217,11 +217,24 @@ def list_heartbeats(*, session: "Session"):
                           Heartbeats.pid,
                           Heartbeats.thread_name,
                           Heartbeats.updated_at,
-                          Heartbeats.created_at).order_by(Heartbeats.readable,
-                                                          Heartbeats.hostname,
-                                                          Heartbeats.thread_name)
-
-    return query.all()
+                          Heartbeats.created_at,
+                          Heartbeats.payload).order_by(Heartbeats.readable,
+                                                       Heartbeats.hostname,
+                                                       Heartbeats.thread_name)
+    result = query.all()
+    json_result = []
+    for r in range(len(result)):
```

Comment:
Better do something like https://github.com/rucio/rucio/blob/b0f7064bc4b07f5be704d656bacdd96e7a03426b/lib/rucio/core/rule.py#L1173-L1776 which stays generic in case we add columns at some point in the future.